compiler/quilt/tools/command.py

```diff
@@ -510,12 +511,12 @@ def _log(team, **kwargs):
     if cfg.get('disable_analytics'):
         return
 
-    session = _get_session(team)
+    try:
+        session = _get_session(team, timeout=LOG_TIMEOUT)
```

Comment:
You also need to modify the `finally` block to check if `session` is set. Anyways, make sure to actually test the timeouts - just set it to 0.001 or something.

Reply:
Tested and works